### ISSUE TITLE
Augmentations not in [0..1]?

### ISSUE THREAD

**Issue by lukasHoel** [hanhung/PureCLIPNeRF#5]
Hi,

thanks for releasing this great project!

After applying the 3 augmentations, you then transform the image into the range suitable for CLIP encoding:
https://github.com/hanhung/PureCLIPNeRF/blob/64c4aea37cfa12a871b7e5cc0d02a8df8a25d2bd/train_imp.py#L413

However `rgb` is not in the `[0...1]` range before applying the transformation, rather I get min/max values like these:

`Min: -0.2289 Max: 1.0150`
`Min: -0.1300 Max: 2.3532`

Is it intended to not clamp or normalize `rgb` back into `[0...1]` range before applying the clip transformation?
I got worse looking results when manually normalizing to `[0...1]` after the augmentations.

**Comment by hanhung** [comment]
Yes, we have also observed this in our experiments. The `rgb` is no longer in the range of `[0...1]` because of the color augmentations in DiffAugment. While we have also tried re-normalizing the image after augmentations it sometimes made the results worse. This is probably due to the re-normalization back to `[0...1]` causing information loss to the image. So we only apply the normalizations of the mean and std used in CLIP.
